src/component/filterDimensionViewer.js

```diff
@@ -9,11 +9,11 @@ export default {
   init({ focusNode, value }, { update }) {
     const baseDimension = focusNode.out(ssz.baseDimension)
     const expectedDimension = createFilterDimension(baseDimension, focusNode)
-    const expectedPath = expectedDimension.out(view.from).out(view.path)
+    const expectedPath = pathSparql(expectedDimension.out(view.from).out(view.path))
```

Comment:
Why that name, pathSparql?

Reply:
Yea, poor choice.

No need to be short either. How about `getSparqlStringForPath`?

Reply:
sure :)